Formula/kyma-cli.rb

```diff
@@ -0,0 +1,38 @@
+class KymaCli < Formula
+  desc "Kyma command-line interface"
+  homepage "https://kyma-project.io"
+  url "https://github.com/kyma-project/cli.git",
+      :tag      => "v0.7.0",
+      :revision => "07d1041389d1cb0b8a9e69cda40e3afa762bbc72"
+  head "https://github.com/kyma-project/cli.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    bin_path = buildpath/"src/github.com/kyma-project/cli/"
+    bin_path.install Dir["*"]
+
+    cd bin_path do
+      system "make", "resolve"
+      system "make", "build"
+      bin.install "bin/kyma-darwin"
+      mv bin/"kyma-darwin", bin/"kyma"
+    end
+  end
+
+  def caveats
+    <<~EOS
+      Happy Kyma-ing! Type \033[1;31mkyma help\033[0m to start
+    EOS
+  end
+
+  test do
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.